egs2/slurp_entity/asr1/local/convert_to_entity_file.py

```diff
@@ -9,12 +9,15 @@ def generate_entity_file(line_arr, output_file="result_test.json"):
     for line in line_arr:
         scenario = line.strip().split("\t")[0].split("_")[0]
         action = "_".join(line.strip().split("\t")[0].split()[0].split("_")[1:])
+        if "▁" not in line:
```

Comment:
Hey @akreal, What is the purpose of this line? 
I prefer that any changes in the data preparation step should be made using arguments so that previously run experiments are still compatible. 
Can you pass an argument to this file and do these steps only while using Hugging Face Tokenizer?